contracts/alliance-nft-collection/src/contract/query.rs

```diff
@@ -119,9 +121,10 @@ fn humanize_approval(approval: &BaseApproval) -> Approval {
 
 // Given the querier and the contract address
 // return the balance of the contract
-pub fn try_query_contract_balance(querier: QuerierWrapper, contract_addr: &Addr) -> Result<Uint128, ContractError> {
-    let contract_balance = querier
-        .query_balance(contract_addr, ALLOWED_DENOM)?
-        .amount;
+pub fn try_query_contract_balance(
+    querier: QuerierWrapper,
+    contract_addr: &Addr,
+) -> Result<Uint128, ContractError> {
+    let contract_balance = querier.query_balance(contract_addr, ALLOWED_DENOM)?.amount;
```

Comment:
Shouldn't this return the balance of ampLUNA instead of Luna?

Reply:
dead code -> removed. Wonder why it isnt being reported by cargo check / cargo clippy

ha... now I know. I will set it up for the workspace to automatically run cargo clippy. Instead of only cargo check. This will visualize these kind of things directly in the source code...